frontend/src/pages/AccountSend.vue

```diff
@@ -806,11 +806,14 @@ function useSendForm() {
   // Validators
   async function isValidId(val: string | undefined) {
     // Return true if nothing is provided
-    checkConfusables();
     if (!val) return true;
 
     // Check if recipient ID is valid
     try {
+      // Check if confusable chars in string, throws with warning if so
+      checkConfusables(val);
```

Comment:
I think this change causes a confusable character to show an error on the frontend rather than a warning. This will prevent users from sending to users that have confusable characters in their ENS name. @garyghayrat is this the behavior we want?

Reply:
> I think this change causes a confusable character to show an error on the frontend rather than a warning. This will prevent users from sending to users that have confusable characters in their ENS name. @garyghayrat is this the behavior we want?

Ah... I see... looks like I've made this too restrictive.. thanks.. I'll fix

Reply:
I'd say warning is sufficient.

Reply:
Restored to warning.. with a new warning element associated with each batch send entry: [34740c2](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/34740c2777b044c67badac8536a056d80aadd2ca)